setup.cfg

```diff
@@ -41,6 +42,7 @@ testing =
     pytest>=3.6,<4
     pytest-cov~=2.8
     pytest-regressions
+    sphinx-external-toc==0.1.0a8
```

Comment:
@AakashGfude this PR should be reviewed against this `version` of `sphinx-external-toc`